doc/overview.rst

```diff
@@ -153,6 +161,9 @@ n_jobs_resample : int | str
 ``preprocessing: pre-SSS bads``: Automatic bad channel detection
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+mf_prebad : dict
+    Dict with subject keys, with each value being a list of int of bad
+    MEG channels (e.g., ``[121, 1743]``).
```

Comment:
oh, this is different from the prebad text file, right?  Where you said `MEG121` not just `121`. Or am I reading this wrong?

Reply:
Actually maybe this is a good time to abandon this number format. It's partly to save work with MaxFilter (which is being replaced anyway), but also partly to save typing `MEG` over and over again. But if you think it's cleaner to define them all as actual channel names like `['MEG0121', 'MEG1743']` instead of `[121, 1743]` we can do that here.

Reply:
should probably ask @nordme and @mdclarke whether they prefer `121` vs `MEG121` here. I don't have a strong opinion.

My only maybe relevant thought is if mnefun were ever used with a different MEG manufacturer with different channel name conventions, would that require mnefun code changes to work. If we can avoid that by how we act now, it seems good to me.

Reply:
I think using `MEG` makes sense to stay consistent and for the reason Dan listed above.